libmscore/noteevent.cpp

```diff
@@ -41,9 +41,12 @@ void NoteEvent::read(XmlReader& e)
 void NoteEvent::write(XmlWriter& xml) const
       {
       xml.stag("Event");
-      xml.tag("pitch", _pitch);
-      xml.tag("ontime", _ontime);
-      xml.tag("len", _len);
+      if (_pitch != 0)
+            xml.tag("pitch", _pitch);
```

Comment:
those `if`s are not needed it using `xml.tag()`'s 3rd agrument

      xml.tag("pitch", _pitch, 0);
      xml.tag("ontime", _ontime, 0);
      xml.tag("len", _len, NOTE_LENGTH);

Reply:
Oh